docs/installation/frameworks/vuejs-v3.md

```diff
@@ -235,15 +235,59 @@ Finally, import the build in your application:
 
 ## Using CKEditor from source
 
-Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}.
+Integrating the rich text editor from source allows you to use the full power of the {@link framework/index CKEditor 5 Framework}. You have two options regarding building your application: Vite or Webpack.
+
+### Vite
+
+This guide assumes that you use `create-vue` as your boilerplate. To get started with Vite and Vue, run the command below.
+
+```bash
+npm init vue@latest ckeditor5-vue-example
+```
+
+This command will install and execute `create-vue`, the official project scaffolding tool for Vue. It will also allow you to customize your project, for example, by adding Typescript. Choose your preferred options.
+
+#### Installing necessary packages
+
+You need two packages to use CKEditor from source with Vue and Vite: the official Vue component and the Vite plugin.
```

Comment:
Same case as in the Vue2 guide, here we mention two packages, but then we ask to install 3 packages.